src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceWorkExperienceReferences.vue

```diff
@@ -195,6 +203,17 @@ export default defineComponent({
     newClientId() {
       return Object.keys(this.modelValue).length + 1;
     },
+    duplicateCharacterReference() {
+      const check = Object.values(this.modelValue).some((workExperienceReference: Components.Schemas.WorkExperienceReference) => {
+        const characterReferenceKey = this.wizardStore.wizardConfig.steps.characterReferences.form.inputs.characterReferences.id;
+        return (
+          workExperienceReference.firstName === this.wizardStore.wizardData?.[characterReferenceKey][0].firstName &&
```

Comment:
Do we need to safely check that we have a character reference? Just making sure this was testing for errors when we don't have a character reference

Reply:
Looks like this was handled below... `wizardStore.wizardData[wizardStore.wizardConfig.steps.characterReferences.form.inputs.characterReferences.id]?.[0]?.emailAddress`

Reply:
Yes we do need it nice catch!